distribution/docs/src/main/resources/content/_configuring/configuring-from-import-intro-contents.adoc

```diff
@@ -1,14 +1,29 @@
-:title: Configuration Export/Import Intro
+:title: Exporting/Importing Configuration Settings
 :type: configuringImport
 :status: published
 :summary: Export a current ${branding} configuration and use it to restore the system to the same state on a brand new installation.
 :order: 00
 
-The Configuration Export/Import capability allows administrators to export the current ${branding} configuration and use it to restore the same state on a brand new installation or when adding a second node to a highly available deployment.
+==== Exporting/Importing Configuration Settings
+
+The Configuration Export/Import capability allows administrators to export the current ${branding} configuration and use it to restore the same state on a brand new installation or to add a second node to a highly available deployment.
+
+To export the current configuration settings:
+. Run the `migration:export` command from the ${command-console}.
+. Copy the `exported-${project.version}.zip` file from the `exported` (unless specified otherwise when invoking the `migration:export`) directory underneath `<${branding}_HOME>` to a secure place.
+
+To import previously exported configuration settings:
+. Step through the installation process in the exact same location as it was on the exported system. The newly installed ${branding}.
```

Comment:
Sentence fragment? Should we explain why the location needs to match?

Reply:
I removed the leftover fragment and add a bit more info. However, I am not sure I want to go in details as to why it must be in the exact same location.